### DISCUSSION TITLE
"Application error: a client-side exception has occurred (see the browser console for more information)."

### DISCUSSION THREAD

**Discussion by fifthshot** [vercel/next.js#57830] [Help]
### Summary

Had a developer build this site, no longer in contact with him. Is there any way I can fix this issue with little coding knowledge or will I have to reach out to another developer to try and fix this issue? Site won't load at all and went into Console: 

A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred
ee @ main-b13111636bafeb66.js:1

Not sure if below code will help...

### Additional information

```js
}
        ;
        function ee(e) {
            let {App: t, err: l} = e;
            return console.error(l),
            console.error("A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred"),
            i.loadPage("/_error").then(n=>{
                let {page: a, styleSheets: o} = n;
                return (null == s ? void 0 : s.Component) === a ? Promise.resolve().then(()=>m._(r(3499))).then(n=>Promise.resolve().then(()=>m._(r(5035))).then(r=>(t = r.default,
                e.App = t,
                n))).then(e=>({
                    ErrorComponent: e.default,
                    styleSheets: []
                })) : {
                    ErrorComponent: a,
                    styleSheets: o
                }
            }
            ).then(r=>{
                var i;
                let {ErrorComponent: u, styleSheets: s} = r
                  , c = Z(t)
                  , f = {
                    Component: u,
                    AppTree: c,
                    router: n,
                    ctx: {
                        err: l,
                        pathname: a.page,
                        query: a.query,
                        asPath: o,
                        AppTree: c
                    }
                };
                return Promise.resolve((null == (i = e.props) ? void 0 : i.err) ? e.props : (0,
                j.loadGetInitialProps)(t, f)).then(t=>eu({
                    ...e,
                    err: l,
                    Component: u,
                    styleSheets: s,
                    props: t
                }))
            }
```

**Comment by icyJoseph** [comment]
Typically this happens when the entire app crashes down. Are there any other logs right before the log that points to that piece of code?

**Comment by fifthshot** [comment]
<img width="546" alt="Screenshot 2023-10-31 at 6 05 27 PM" src="https://github.com/vercel/next.js/assets/149530353/5d04d657-55fc-49f3-80c9-9fe7adb50a33">
<img width="795" alt="Screenshot 2023-10-31 at 6 07 16 PM" src="https://github.com/vercel/next.js/assets/149530353/c5095417-5d19-4bab-9e00-7904a5043d37">
<img width="798" alt="Screenshot 2023-10-31 at 6 07 09 PM" src="https://github.com/vercel/next.js/assets/149530353/43c53dfe-d3b5-4176-8378-fdd6c5694493">

Not sure if these screenshots help...

**Reply by icyJoseph** [reply]
Well sort of. I can speculate that the api call for players is malfunctioning, which means that a list that's expected at Home, is now undefined, and so it can't be mapped over anymore. Calling the map method on undefined crashes the entire thing.

What's that player API endpoint doing? Perhaps it needs a new API key to communicate to a 3rd party service?

Of course assuming that things just worked and have suddenly just stopped working.

**Reply by fifthshot** [reply]
Yes, things just suddenly stopped working. Really, really appreciate the help!

How do I find out what the API endpoint is doing? Only thing that comes up in Souce Code is:

{"message":"Internal Server Error"}

**Reply by icyJoseph** [reply]
Do you not have have access to the repository containing the code? That's where you have to look. 

Alternatively, you can look at the Vercel Dashboard, and see if there are any Environment Variables there. Perhaps you need to regenerate an API key for a 3rd party service?

**Comment by fifthshot** [comment]
Don't have access to the repository, guess I have to reach out to the
original developer again.

Thanks for the help!
